BSB_LAN/BSB_LAN.ino

```diff
@@ -529,62 +529,63 @@ EEPROMClass EEPROM_ESP("nvs", EEPROM_SIZE);
 #include <Wire.h>
 #include "src/BlueDot_BME280/BlueDot_BME280.h"
 BlueDot_BME280 *bme;  //Set 2 if you need two sensors.
+//Multiplexor TCA9548A (if presented) address on I2C bus
+#define TCA9548A_ADDR 0x70
 #endif
 
 bool client_flag = false;
 #ifdef WIFI
   #ifdef ESP32
-#include <WiFi.h>
-bool localAP = false;
-unsigned long localAPtimeout = millis();
+    #include <WiFi.h>
+    bool localAP = false;
+    unsigned long localAPtimeout = millis();
   #else
-#include "src/WiFiSpi/src/WiFiSpi.h"
-using ComServer = WiFiSpiServer;
-using ComClient = WiFiSpiClient;
-#define WiFi WiFiSpi
+    #include "src/WiFiSpi/src/WiFiSpi.h"
+    using ComServer = WiFiSpiServer;
+    using ComClient = WiFiSpiClient;
+    #define WiFi WiFiSpi
   #endif
 #else
-
   #ifdef ESP32
-#define ETH_CLK_MODE ETH_CLOCK_GPIO17_OUT
-#define ETH_PHY_POWER 12
-#include <ETH.h>
-
-class Eth : public ETHClass {
-public:
-    int maintain(void) const { return 0;} ; // handled internally
-    void begin(uint8_t *mac, IPAddress ip, IPAddress dnsserver, IPAddress gateway, IPAddress subnet) {
-      ETHClass::begin(ETH_PHY_ADDR, ETH_PHY_POWER, ETH_PHY_MDC, ETH_PHY_MDIO, ETH_PHY_TYPE, ETH_CLK_MODE);
-      config(ip, gateway, subnet, dnsserver, dnsserver); //Static
-    }
-    void begin(uint8_t *mac) {
-      ETHClass::begin(ETH_PHY_ADDR, ETH_PHY_POWER, ETH_PHY_MDC, ETH_PHY_MDIO, ETH_PHY_TYPE, ETH_CLK_MODE);
-    }
-};
+    //#define ETH_CLK_MODE ETH_CLOCK_GPIO17_OUT
+    //#define ETH_PHY_POWER 12
+    #include <ETH.h>
+
+    class Eth : public ETHClass {
+    public:
+        int maintain(void) const { return 0;} ; // handled internally
+        void begin(uint8_t *mac, IPAddress ip, IPAddress dnsserver, IPAddress gateway, IPAddress subnet) {
+          begin(mac);
```

Comment:
Why did you remove the ETHClass::begin here? At least here, it still works even if the ETH_CLK_MODE lines above are commented out because then the defaults will be used.

Reply:
Don't panic! :) it is just short call of ETHClass::begin(ETH_PHY_ADDR, ETH_PHY_POWER, ETH_PHY_MDC, ETH_PHY_MDIO, ETH_PHY_TYPE, ETH_CLK_MODE);